scss/_variables.scss

```diff
@@ -405,7 +405,7 @@ $aspect-ratios: (
   "1x1": 100%,
   "4x3": calc(3 / 4 * 100%),
   "16x9": calc(9 / 16 * 100%),
-  "21x9": calc(9 / 21 * 100%)
+  "21x9": calc(9 * .51 * 100%)
```

Comment:
I think this is a find & replace mistake and `9 / 21` should not be converted

Reply:
looks like `/ 2`1 got replaced by `* .5`1